site-src/guides/multiple-ns.md

```diff
@@ -1,38 +1,65 @@
+
+## Resource Deployment
+
+The infrastructure team deploys the `shared-gateway` Gateway into the `infra` Namespace. 
+
+```yaml
+{% include 'cross-namespace-routing/gateway.yaml' %}
 ```
-{% include 'routes-in-multiple-namespaces.yaml' %}
+
+A couple notes about this Gateway:
+
+- It is matching for the `foo.example.com` domain. This is configured on the Gateway so that each HTTPRoute does not also have to configure hostname matching, since they are all using the same domain.
+- The Gateway is configured for HTTPS and references the `foo-example-com` Secret. This allows the certificate to be managed centrally for all applications which are using this Gateway.
+- `routes.Namespaces.from: All` is the configuration to allow routes from any Namespace to use this Gateway. This impacts how routes are able to bind to this Gateway.
+
+The store team deploys their route for the `store` Service in the `store` Namespace:
+
+```yaml
+{% include 'cross-namespace-routing/store-route.yaml' %}
+```
+
+This Route has straightforward routng logic as it just matches for `/store` traffic which it sends to the `store` Service. The following snippet of the [`gateways` field](/references/spec/#networking.x-k8s.io/v1alpha1.RouteGateways) controlls which Gateways this Route can bind to:
+
+```yaml
+  gateways:
+    allow: FromList
+    gatewayRefs:
+    - name: shared-gateway
+      namespace: infra
 ```
 
-Please note that this guide illustrates this feature for HTTPRoute resource
-only as an example. The same can be accomplished with other route types as
-well.
+`gateways.allow` can be configured for Gateways in the same Namespace as the Route (the default), all Gateways, or a list of specific Gateways. In this example the store and site teams decide to reference a specific Gateway. This is the least permissive choice which ensures that other Gateways in the cluster (perhaps created in the future at some point) will not bind with these Routes. If cluster administrators have full control over how Gateways are deployed in a cluster then a more permissive binding option could be configured on Routes. The less permissive the Gateway selection is, the less that application owners need to know about which Gateways are deployed. 
+
+The site team now deploys Routes for their applications. They deploy two HTTPRoutes into the `site` Namespace:
+
+- `httproute/home` which acts as a default routing rule, matching for all traffic to `foo.example.com/*` that is not matched by an existing routing rule and routing it to `service/home`
```

Comment:
This `type/name` form is new for our docs. I'd recommend avoiding it and using "HTTPRoute `home`" style for consistency purposes.